File: app/src/main/java/uk/co/ianfield/devstat/model/StatItem.java
package uk.co.ianfield.devstat.model;

/**
 * Created by Ian Field on 20/02/2014.
 */
public class StatItem {
    String title;
    String info;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getInfo() {
        return info;
    }

    public void setInfo(String info) {
        this.info = info;
    }

    public String toString() {
        return String.format("%s: %s", getTitle(), getInfo());
    }
}

File: app/src/main/java/uk/co/ianfield/devstat/model/StatItem.kt
package uk.co.ianfield.devstat.model

/**
 * Created by Ian Field on 20/02/2014.
 */
open class StatItem {
    lateinit var title: String
    lateinit var info: String

    override fun toString(): String {
        return String.format("%s: %s", title, info)
    }
}
